src/types/redis_string.cc

```diff
@@ -371,28 +371,33 @@ rocksdb::Status String::MSet(const std::vector<StringPair> &pairs, uint64_t ttl,
     expire = now + ttl;
   }
 
-  // Data race, key string maybe overwrite by other key while didn't lock the key here,
+  // Data race, key string maybe overwrite by other key while didn't lock the keys here,
   // to improve the set performance
   std::string ns_key;
+  std::optional<MultiLockGuard> guard;
+  if (lock) {
+    std::vector<std::string> lock_keys;
+    lock_keys.reserve(pairs.size());
+    for (StringPair pair : pairs) {
```

Comment:
```suggestion
    for (const StringPair& pair : pairs) {
```

In previous patch, pair need to be copied, but here we doesn't need that.

Reply:
thanks.